CAIPs/caip-275.md

```diff
@@ -199,91 +202,82 @@ However, this configuration is not recommended as having multiple wallets genera
 ```
 
 Using this configuration, the application should provide the QR code and [Wallet Connect][] URI for the user to connect with his wallet from any place.
-Such configuration can be useful when the user has their wallet in their mobile device and also on another platform such as a web wallet or desktop application that can receive the QR code or [Wallet Connect][] URI in any way and they want to decide which one to use at connection time.
+Such configuration can be useful when the user has their wallet in their mobile device and also on another platform such as a web wallet or desktop application that can receive the QR code or [Wallet Connect][] URI in any way, and they want to decide which one to use at connection time.
 
-This configuration will never be skipped as it does not impose any platform and requires the user action of connecting from the wallet making this a great last case in the `authFlows` array and behaving as the default when no other authentication flow is useful.
+This configuration should never be skipped as it does not impose any platform and requires the user action of connecting from the wallet making this a great last case in the `authFlows` array and behaving as the default when no other authentication flow is useful.
+The only requirement for this configuration to work is that the application supports [Wallet Connect][].
 
-## Self hosting authentication flows
+## Self-hosting authentication flows
 
 In case the user does not trust any authentication flow provider to store their info, they can easily host it themselves using a public serverless function and saving its URL in the ENS `authenticator` record.
```

Comment:
serverless like service worker URI? or a static gitpages website?  I would probably provide an appendix (for each, if both work) because this is a little hard to picture and people deciding whether or not to implement this would probably need to know how foolproof an automation of this self-hosting option is before commiting 😄